src/bigquery.ts

```diff
@@ -376,12 +376,34 @@ export class BigQuery extends common.Service {
    *
    * @param {object} schema
    * @param {array} rows
+   * @param {string} selectedFields List of fields to return (comma-separated).
```

Comment:
Can we accept this as an array of strings instead?

Reply:
Done!!